holopy/core/io/io.py

```diff
@@ -380,6 +380,96 @@ def save_image(filename, im, scaling='auto', depth=8):
         pilimage.fromarray(im).save(filename)
 
 
+def save_images(filenames, ims, scaling='auto', depth=8):
+    """
+    Saves a volume as separate images (think of reconstruction volumes).
+
+    Parameters
+    ----------
+    ims : ndarray or :class:`holopy.image.Image`
```

Comment:
The function argument takes parameters in the order (filenames, ims), but the docstring provides them in order (ims, filenames). Can you switch the order to be consistent with the function argument order? I see that you just copied the docstring for save_image, which has this order. So if you think this is out of the scope of this PR, let me know and I'll add it to a TODO list for later.

Reply:
Done!